README.md

```diff
@@ -25,33 +25,35 @@
 </div>
 
 ### Contents:
-  - [Categories](#categories)
```

Comment:
Why did you refactor the code? Was wrong?

Reply:
i dont remember to be honest , maybe i faced some issue, i want to contribute , accept my pull request please thanks.

Reply:
But if you want to add just one profile, should be changed just one line. Refactor should be in other commit, even PR.